### ISSUE TITLE
Ocean Jasper Fluted Column texture error

### ISSUE THREAD

**Issue by NotAVerb** [GlobbyPotato/Rockhounding_Rocks#6]
The ocean jasper fluted column has the basanite fluted column texture in JEI and in the world. Connecting the column to another column fixes the texture. By itself the column is indistinguishable from basanite.

![2018-03-11_21 51 38](https://user-images.githubusercontent.com/30223331/37261953-f0289cd0-2597-11e8-9c71-3cea9215b537.png)
V 3.07

**Comment by GlobbyPotato** [comment]
yep it is just a json texture error. i've spotted a couple already, some others should be there unnoticed, usually they pop out when elongating the column as it reveals the combined parts of the column. since there are too many combinations I check them in groups every update.

**Comment by NotAVerb** [comment]
I haven't found any other broken blocks in 3.08 yet, but I thought I'd leave some feedback about how I don't like the new round doric pillars. I much preferred when they were "blocky".

**Comment by GlobbyPotato** [comment]
I understand the rounder pillar won't really suit much, but it is a kind of compromise I'll take for good. A lot of rocks have a too explicit color pattern. Not every rock works the same way for every shape, pattern or use but that made this particular column look horrible on too many rocks than I expected so I made it less brutal for eyes. No other rounder blocks are expected indeed. Been cleaning some mapping even more after the latest release to make them more fluid.

I got the time to place down all the blocks at once and hopefully fixed them all, including some extra bugs. If nothing arises will close this issue for now to fit the released versions.
